src/avram/save_operation.cr

```diff
@@ -33,8 +35,12 @@ abstract class Avram::SaveOperation(T)
 
   @record : T?
   @params : Avram::Paramable
+
   getter :record, :params
+
   property save_status : SaveStatus = SaveStatus::Unperformed
+  property record
```

Comment:
Let's move the `getter :record` to this so we're not defining that getter twice.